src/styles/main.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@import 'blocks/utils/variables';
+@import 'blocks/header';
+@import 'blocks/nav';
+@import 'blocks/card';
+@import 'blocks/stars';
+@import 'blocks/container';
+
+* {
   margin: 0;
-}
+  box-sizing: border-box;
+ }
+
+ body {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+ }
+
```

Comment:
It's better to move general styles to a separate file and import that file before other styles